config/puma.rb

```diff
@@ -32,7 +32,7 @@
 # Workers do not work on JRuby or Windows (both of which do not support
 # processes).
 #
-# workers ENV.fetch("WEB_CONCURRENCY") { 2 }
+workers ENV.fetch("WEB_CONCURRENCY") { 1 }
```

Comment:
Style/RedundantFetchBlock: Use `fetch("WEB_CONCURRENCY", 1)` instead of `fetch("WEB_CONCURRENCY") { 1 }`. (https://github.com/JuanitoFatas/fast-ruby#hashfetch-with-argument-vs-hashfetch--block-code)